crypto/fipsmodule/kdf/internal.h

```diff
@@ -12,8 +12,8 @@
  * OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
  * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE. */
 
-#ifndef OPENSSL_HEADER_CRYPTO_FIPSMODULE_TLS_INTERNAL_H
```

Comment:
Are you planning on putting other internal KDF functions in this header file? Otherwise I'd like to keep the name and this macro related to TLS as that's the only function in here.

Reply:
Yeah, I was planning on putting the SSH KDF I'm adding in CryptoAlg-1226 in here as well.